modules/core/src/main/java/org/apache/ignite/internal/processors/cache/query/GridCacheQueryManager.java

```diff
@@ -1349,13 +1349,7 @@ protected void runQuery(GridCacheQueryInfo qryInfo) {
                             if (!finished)
                                 rmvIter = false;
 
-                            if (!qryInfo.allPages())
-                                return;
-
-                            data = new ArrayList<>(pageSize);
-
-                            if (stop)
-                                break; // while
```

Comment:
Setting `pageSent` in this block now useless

Reply:
`PageSent` removed from this block.